packages/core/src/plugin-state/list.ts

```diff
@@ -77,6 +78,22 @@ export function list<S, T = S, R = unknown>(
           onChange(items => {
             return R.remove(index, 1, items)
           })
+        },
+        move(f: number, t: number) {
+          onChange(items => {
+            // FIXME: use 'return R.move(from, to, items)' in next ramda version
```

Comment:
?

Reply:
Ramda hasn't released the `move` function yet, though it is already in their docs (https://ramdajs.com/docs/#move). I just copied the implementation from their PR (and simplified a bit because we don't need handling of negative indices)
https://github.com/ramda/ramda/blob/271b044c5b18329d2de2d5a786123ab7ca34d11e/source/move.js#L20-L34

But I think we should switch to `R.move` as soon as its there

Reply:
@Knorrke Do you issue a ticket to track this?

Reply:
Sure, will do.

Reply:
Actually nvm, I just noticed they added it in their latest patch version, which I missed. So I'll just upgrade to that and replace this code :-)